lib/commands/start.mjs

```diff
@@ -12,17 +12,23 @@ import {decoratedError, ErrorType} from "../browserup_errors.mjs";
 import {WebConsoleClient} from "../services/webconsole_client.mjs";
 import {calculateSHA256Hash} from "../utils/hash_utils.mjs";
 import FormData from "form-data";
+import {getDirectorySize} from "../commands/detect_dir_size.mjs"
+import {isSystemDirectory} from "../commands/path_detect.mjs";
```

Comment:
We should be consistent with spacing here and follow JS conventions. Look at other imports in other programs to figure out if the curly braces get a space after/before them in module imports.